org.eclipse.scout.rt.dataobject/src/main/java/org/eclipse/scout/rt/dataobject/migration/IDoValueMigrationHandler.java

```diff
@@ -138,6 +139,9 @@
 @ApplicationScoped
 public interface IDoValueMigrationHandler<T> {
 
+  double UNTYPED_PRIMARY_SORT_ORDER = 1000.0;
```

Comment:
add javadoc indicating default usage of these constants? similar like javadoc comment for primarySortOrder() method